typed-racket-lib/typed-racket/types/subtype.rkt

```diff
@@ -804,6 +810,17 @@
      ;; tvars are equal if they are the same variable
      [(F: var2) (eq? var1 var2)]
      [_ (continue<: A t1 t2 obj)])]
+  [(case: FlatCon (FlatCon: t1-pre t1-post))
+   (match t2
+     [(FlatCon: t2-pre t2-post)
+      (subtype-seq A
+                   (subtype* A t2-pre t1-pre)
+                   (subtype* A t1-post t2-post))]
```

Comment:
same issues w/ `subtype-seq` (i.e. erase the `A`'s after each `subtype*` in the `subtype-seq`